advance.cloudbuild.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright 2021 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+
+- id: 'Download dependencies'
+  name: golang
+  args: ['go', 'mod', 'download']
+
+- id: 'Install linting tools'
+  name: golang
+  args: ['go', 'install', 'golang.org/x/tools/cmd/goimports@latest']
+
+- id: 'Lint'
```

Comment:
Is this linting or fixing lint?

Reply:
Whoops, yeah the intention was lint. Updated.